en/release-notes/upgrade-guide-v2.1.md

```diff
@@ -14,67 +14,67 @@ This is not the first time __Cocos__ has upgraded the engine from 2D to 3D. A fe
 
 __v2.0__ and __v2.1__ will grow together over a period of time.__v2.0__ will continue to be maintained, adding more performance optimizations and bug fixes. We will soon be beta testing v2.0.7. In v2.0.7 the rendering performance of the engine has been optimized, especially **Spine** and **DragonBones**. Also, the performance has been greatly improved.
 
-**v2.1 iterations will be more aggressive**
+### v2.1 iterations will be more aggressive
```

Comment:
下面截图中的两个标题应该是同一个层级关系，如果 [v2.1 iterations will be more aggressive] 要用第三级目录的话，我觉得上一个关于 2.0 的也应该用第三级目录～
![image](https://user-images.githubusercontent.com/19702899/49566027-41283500-f964-11e8-9232-94cba34479cd.png)

Reply:
I see, I didn't view it what way because it was so long. I thought it looked like a "note: ". Changed now and I edited the text to be shorter since we say this in the body text too.